src/shared/forms/controllers/PasswordInputController.test.tsx

```diff
@@ -41,24 +41,46 @@ describe('<PasswordInputController />', () => {
     '1 Chiffre',
     '1 Caractère spécial (!@#$%^&*...)',
   ])('should show password validation rules when asked', (rules) => {
-    renderPasswordInputController({ withSecurityRules: true })
+    renderPasswordInputController({ withSecurityRules: true, securityRulesAlwaysVisible: true })
 
     expect(screen.getByText(rules)).toBeTruthy()
   })
 
   it('should not show password validation rules by default', () => {
-    renderPasswordInputController({})
+    renderPasswordInputController({ withSecurityRules: true })
+
+    expect(screen.queryByText('12 Caractères')).toBeFalsy()
```

Comment:
Je me demande si on devrait pas faire un it.each et bien tester que toutes les règles ne sont bien pas présente 🧐